L1Trigger/TrackFindingTracklet/python/ProducerKF_cfi.py

```diff
@@ -3,17 +3,17 @@
 TrackFindingTrackletProducerKF_params = cms.PSet (
 
   InputTag             = cms.InputTag( "TTTracksFromTrackletEmulation", "Level1TTTracks"), #
-  InputTagDTC          = cms.InputTag( "TrackerDTCProducer", "StubAccepted"),                    #
-  LabelKFin            = cms.string  ( "TrackFindingTrackletProducerKFin"  ),                    #
-  LabelKF              = cms.string  ( "TrackFindingTrackletProducerKF"    ),                    #
-  LabelTT              = cms.string  ( "TrackFindingTrackletProducerTT"    ),                    #
-  LabelAS              = cms.string  ( "TrackFindingTrackletProducerAS"    ),                    #
-  LabelKFout           = cms.string  ( "TrackFindingTrackletProducerKFout" ),                    #
-  BranchAcceptedStubs  = cms.string  ( "StubAccepted"  ),                                        #
-  BranchAcceptedTracks = cms.string  ( "TrackAccepted" ),                                        #
-  BranchLostStubs      = cms.string  ( "StubLost"      ),                                        #
-  BranchLostTracks     = cms.string  ( "TrackLost"     ),                                        #
-  CheckHistory         = cms.bool    ( False ),                                                  # checks if input sample production is configured as current process
-  EnableTruncation     = cms.bool    ( True  ),                                                  # enable emulation of truncation, lost stubs are filled in BranchLost
+  InputTagDTC          = cms.InputTag( "TrackerDTCProducer", "StubAccepted"),              #
```

Comment:
It's wierd that the KF needs to know the name of the DTC collection, when the KF doesn't read directly the DTC.

Reply:
ProdcucerKF_cfi is used by all of mine Producer, namely ProducerKF, ProducerKFout, ProducerKFin, ProducerTT, ProducerIrin, ProducerTBout. it is named ProducerKF_cfi because you requested that name change from original Producer_cfi in the past.

Reply:
Ah! Perhaps Producer_cfi was best after all. Sorry ...

Reply:
done